velox/connectors/hive/storage_adapters/s3fs/RegisterS3FileSystem.cpp

```diff
@@ -15,13 +15,12 @@
  */
 
 #ifdef VELOX_ENABLE_S3
-#include "velox/connectors/hive/HiveConfig.h" // @manual
+#include "velox/connectors/hive/S3Config.h" // @manual
```

Comment:
can these S3-specific configs also live under connectors/hive/storage_adapters/s3fs?

Reply:
I thought about this earlier.
The HiveConfig would be public API and is independent of enabling VELOX_ENABLE_S3 which s3fs depends upon.
Gluten code for example uses the HiveConfig (S3Config with this change) to translate from Spark config to Velox config.

Reply:
I think VELOX_ENABLE_S3 needs to be enabled for the S3Filesystem registration. So I am overthinking here. I will move the config.